HTMLSerializer.js

```diff
@@ -122,8 +160,8 @@ var HTMLSerializer = class {
     var win = element.ownerDocument.defaultView;
     var style = win.getComputedStyle(element, null).cssText;
     var windowDepth = this.windowDepth(win);
-    style = style.replace(/"/g, this.escapedQuote(windowDepth+1));
-    var quote = this.escapedQuote(windowDepth);
+    style = style.replace(/"/g, this.escapedCharacter('"', windowDepth+1));
```

Comment:
Using "windowDepth+1" for style but just "windowDepth" below was a little confusing to me at first since they are in the same window. WDYT of using "nestingDepth" for the local variable name (i.e., "var nestingDepth = this.windowDepth(win);")? Similarly in processText.